SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -33,10 +33,10 @@ \chapter{Configuring and Building ITK}
 %% to ensure that support is maintained.
 %%
 \begin{itemize}
-\item GCC 4.8 and newer
+\item GCC 5.1 and newer
 \item Visual Studio 2015, 2017, 2019, toolsets 14.x, 15.0, 16.0
```

Comment:
VS 2015 might need to be revised. Don't know about the toolsets.

Reply:
> VS 2015 might need to be revised.

Thanks @jhlegarreta It's not strictly necessary to drop VS 2015, as it already supports most of C++14. But then again, it does still miss some C++14 features (specifically [Extended constexpr](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3652.html)).  So it's still to be discussed: https://discourse.itk.org/t/may-we-start-using-c-14-at-the-master-branch/4172/2

Reply:
VS 2015 dropped with my latest amend + [force-push](https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/compare/fc64bda5a4a987a736a710f611e7814ed752b20c..80b8f8913eabd3c4d46a64389e0e1bd442e009da).